src/lib/button/index.ts

```diff
@@ -1 +1,49 @@
+import {ModuleWithProviders, NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {MdRippleModule} from '../core/ripple/ripple';
+import {CompatibilityModule} from '../core/compatibility/compatibility';
+import {
+  MdAnchor,
+  MdButton,
+  MdButtonCssMatStyler,
+  MdFabCssMatStyler,
+  MdIconButtonCssMatStyler,
+  MdMiniFabCssMatStyler,
+  MdRaisedButtonCssMatStyler
+} from './button';
+import {StyleModule} from '../core/style/index';
+
+
 export * from './button';
+
+
+@NgModule({
+  imports: [CommonModule, MdRippleModule, CompatibilityModule, StyleModule],
+  exports: [
+    MdButton, MdAnchor,
+    CompatibilityModule,
+    MdButtonCssMatStyler,
+    MdRaisedButtonCssMatStyler,
+    MdIconButtonCssMatStyler,
+    MdFabCssMatStyler,
+    MdMiniFabCssMatStyler
```

Comment:
It seems like maybe we should export `StyleModule` so you don't have to import it separately when using the `MdButtonModule` alone.